src/Illuminate/Database/Console/WipeCommand.php

```diff
@@ -17,15 +17,6 @@ class WipeCommand extends Command
      */
     protected $name = 'db:wipe';
 
-    /**
-     * The name of the console command.
-     *
-     * This name is used to identify the command during lazy loading.
-     *
-     * @var string|null
-     */
-    protected static $defaultName = 'db:wipe';
-
```

Comment:
Was there a reason you deleted that instead of add `@deprecated` and `AsCommand` Attribute?

Reply:
I checked all other changed commands and I found no missing `@deprecated` or `AsCommand`. I also checked if all command names are correctly copied.

Reply:
@Jubeki nope, I re-added that. Thanks for spotting and reviewing!

Reply:
I don't know why I did the same. ))